SessionUtilitiesKit/General/String+Utilities.swift

```diff
@@ -147,7 +147,7 @@ public extension String {
             case .twoUnits: // 2 units, no localization, short version e.g 1w 1d
                 dateComponentsFormatter.maximumUnitCount = 2
                 dateComponentsFormatter.unitsStyle = .abbreviated
-                dateComponentsFormatter.zeroFormattingBehavior = .dropLeading
+                dateComponentsFormatter.zeroFormattingBehavior = .dropAll // Changed from .dropLeading to also remove trailing 0's ei. 12h 0m -> 12h
```

Comment:
Probably don't need a commend saying it was changed from something else - if you want to indicate that `0` values will be trimmed then you can probably just update the comment next to `case .twoUnits`